core-bundle/src/Resources/contao/helper/functions.php

```diff
@@ -255,7 +255,7 @@ function trimsplit($strPattern, $strString)
  */
 function ampersand($strString, $blnEncode=true)
 {
-	@trigger_error('Using ampersand() has been deprecated and will no longer work in Contao 5.0. Use StringUtil::ampersand() instead.', E_USER_DEPRECATED);
+	trigger_deprecation('contao/core-bundle', '4.0', 'Using "ampersand()" has been deprecated and will no longer work in Contao 5.0. Use "Contao\StringUtil::ampersand()" instead.');
```

Comment:
not '4.0', but '4.10'

Reply:
No, I think the `ampersand()` function has been deprecated since Contao 4.0.

Reply:
Yes, but Contao\StringUtil::ampersand() is only available from version 4.10.
If I'm to use the proposed replacement, then it must also exist, I think.

Reply:
Have a look at the phpdoc:
`* @deprecated Deprecated since Contao 4.10, to be removed in Contao 5.0.`

Reply:
You are right. I am going to fix this.

Reply:
See #2062.